index.js

```diff
@@ -62,10 +62,27 @@ function createFocusTrap(element, userOptions) {
     deactivate: deactivate,
     pause: pause,
     unpause: unpause,
+    updateContainerElements: updateContainerElements,
   };
 
+  updateContainerElements(elements);
+
   return trap;
 
+  function updateContainerElements(containerElements) {
+    var elementsAsArray = [].concat(containerElements).filter(Boolean);
+
+    state.containers = elementsAsArray.map((element) =>
+      typeof element === 'string' ? doc.querySelector(element) : element
+    );
+
+    if (state.active) {
+      updateTabbableNodes();
+    }
+
+    return trap;
```

Comment:
Marking this here (so I don't forget) as something to remove depending on your thoughts on my comment in `index.d.ts`...